examples/caddy2/Caddyfile

```diff
@@ -123,6 +144,8 @@ element.DOMAIN.tld {
                 X-Content-Type-Options "nosniff"
                 # Disallow the site to be rendered within a frame (clickjacking protection)
                 X-Frame-Options "DENY"
```

Comment:
You're not removing `X-Frame-Options` here, but you are removing it from all other places?

Reply:
The reasoning is that Jitsi and Dimension can be framed into Element, but Element isn't framed in anything by default since it is the "root" client app.